src/analysis/special_functions/integrals.lean

```diff
@@ -236,6 +236,9 @@ by simpa using integral_pow 1
 lemma integral_one : ∫ x in a..b, (1 : ℝ) = b - a :=
 by simp only [mul_one, smul_eq_mul, integral_const]
 
+lemma integral_const_on_unit_interval : ∫ x in a..(a + 1), b = b :=
```

Comment:
Is there a better name than "unit_interval" for this? To me, that suggests the interval [0,1], but I also don't have any better suggestions unfortunately

Reply:
`unit_length_interval`? Just starts getting very long in name....